cmd/gitops/add/management-cluster/flag.go

```diff
@@ -0,0 +1,50 @@
+package mcluster
+
+import (
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagDryRun          = "dry-run"
+	flagInterval        = "refresh-interval"
+	flagName            = "name"
+	flagLocalPath       = "local-path"
```

Comment:
If it is possible / convenient to implement I think the local path and dry run flags should be global as they make sense for all sub commands.

For example: `kubectl gs gitops --local-path ~/repos/gitops add mc --name dummy`

Reply:
yup, I've already started to do it, but haven't pushed yet